README.md

```diff
@@ -32,269 +34,51 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 ## Run Deephaven
 
-This section is a quick start guide for running Deephaven from pre-built images.  Almost all users will want to run Deephaven using pre-built images.  It is the easiest way to deploy.  For detailed instructions, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart).
+If you want to use Deephaven, see our [Quick start for Docker guide](https://deephaven.io/core/docs/tutorials/quickstart).
 
-Developers interested in tinkering with and modifying source code should build from the source code. For detailed instructions on how to do this, see [Build and launch Deephaven](https://deephaven.io/core/docs/how-to-guides/launch-build).
+Developers interested in modifying source code should read on for development instructions.
 
-If you are not sure which of the two is right for you, use the pre-built images.
+## Development Instructions
 
 ### Required Dependencies
 
-Running Deephaven requires a few software packages.
-
-| Package        | Version                       | OS           |
-| -------------- | ----------------------------- | ------------ |
-| docker         | ^20.10.8                      | All          |
-| docker-compose | ^1.29.0                       | All          |
-| Windows        | 10 (OS build 20262 or higher) | Only Windows |
-| WSL            | 2                             | Only Windows |
+Developing in Deephaven requires a few software packages.
 
-You can check if these packages are installed and functioning by running:
-```
-docker version
-docker-compose version
-docker run hello-world
-```
+| Package | Version                       | OS           |
+| ------- | ----------------------------- | ------------ |
+| Java    | >=11, <20                     | All          |
+| Python  | >= 3.8                        | All          |
+| Windows | 10 (OS build 20262 or higher) | Only Windows |
+| WSL     | 2                             | Only Windows |
```

Comment:
This dependency list does not match what is on the website.
https://deephaven.io/core/docs/how-to-guides/launch-build/
Is python a hard requirement?  If not, that is not represented in either place.

Reply:
Python isn't a hard requirement. Docker isn't either technically I think.

I'm also not sure the Java version is entirely correct here or the website. I _think_ gradle requires a more specific version to run, but not 100% sure (I had some issues in intellij b/c I had my gradle version set too new)